wifiphisher/pywifiphisher.py

```diff
@@ -1,24 +1,21 @@
 #!/usr/bin/env python2
 # -*- coding: utf-8 -*-
 # pylint: skip-file
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
```

Comment:
Why are we importing these?

Reply:
The motivation is to add this to all the files so that we don't go back to `python2` only features. An example would be the `print` statement so if you wanted to add a `print` statement in the code with this import in place you wouldn't be able to do this:
```python
print "HI"
```
This ensures that we don't regress to `python2` only.